test/Interop/Cxx/class/type-classification-loadable-silgen.swift

```diff
@@ -54,17 +54,17 @@ func pass(s: StructWithSubobjectDefaultedCopyConstructor) {
 
 // CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithPrivateDefaultedCopyConstructor)
 func pass(s: StructWithPrivateDefaultedCopyConstructor) {
-  // CHECK: bb0(%0 : $*StructWithPrivateDefaultedCopyConstructor):
```

Comment:
IMHO we probably shouldn't be importing any of these.

Reply:
Aha! I think I know what's going on here. The record is being cached before this visitor returns nullptr, so when it gets looked up when getting the function type, it looks like it exists. Erasing it from the imported decls map should fix this.